MediaProcessor/src/main.cpp

```diff
@@ -58,4 +58,4 @@ int main(int argc, char* argv[]) {
 
     std::cout << "Video processed successfully: " << processedMediaPath << std::endl;
     return 0;
-}
```

Comment:
please run the clang (with the file available on project root, i.e. .clang-format) for MediaProcessor/src or setup the pre-commit hooks which would do this for you